docs/data/playerStateDescription.md

```diff
@@ -0,0 +1,6 @@
+# PlayerStateDescription
+
+A PlayerStateDescription is a well formed string derived from the [RoomState](./roomState.md) (and consequently the [PlayerState](./PlayerState.md)).
```

Comment:
`./PlayerState.md` should be `./playerState.md` (linux is case sensitive)

Reply:
I think I fixed all of them like this